### PR TITLE
Additional Content

### PR DESCRIPTION
**Pull request by cthepot**

### CODE REVIEW

File: site/content/news/atomic-swaps.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+The crypto world will never be truly decentralised until we find a way to facilitate trustless token exchanges.  
+
+Introduced in 2013 by Tier Nolan, _atomic cross-chain trading_, aka _atomic swaps_ might just be the solution. Thanks to the Lightning network, an off-chain scaling solution for blockchain, users can swap digital currencies instantly and securely.  
+
+It is considered “atomic” because the trade either happens or not, without the space for any discrepancies. 
+
+With this technology breakthrough, users do not need to trust any centralised entity or a peer to swap currencies.  
+
+Let’s illustrate this with a simple an example:
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: a simple example

File: site/content/news/atomic-swaps.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+•	Alice has 1 BTC and wants 20 LTC.
+•	Bob has 20 LTC and wants 1 BTC.
+•	Alice and Bob do not know each other and are afraid of exchange hacks.
+
+Rather than logging on [Coinbase] (https://www.coinbase.com/join/59abb5ab6c9e6700d36656d6) or [Shapeshift] (https://shapeshift.io), Alice and Bob should set up an atomic cross-chain swap to guarantee the transaction occurs on the two networks.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: Incorrect link

File: site/content/news/atomic-swaps.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+1 – Alice and Bob issue a transaction on the Bitcoin and Litecoin blockchain.
+2 – Alice is able to claim the 20 LTC if she reveals a number that she only has access to.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: that **only she** has access to

File: site/content/news/atomic-swaps.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+3 – Similarly, Bob is able to claim his BTC if he reveals the same number.
+4 – When one of them claim their tokens, the other one can do the same.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: claims

File: site/content/news/atomic-swaps.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Let’s illustrate this with a simple an example:
+
+•	Alice has 1 BTC and wants 20 LTC.
+•	Bob has 20 LTC and wants 1 BTC.
+•	Alice and Bob do not know each other and are afraid of exchange hacks.
+
+Rather than logging on [Coinbase] (https://www.coinbase.com/join/59abb5ab6c9e6700d36656d6) or [Shapeshift] (https://shapeshift.io), Alice and Bob should set up an atomic cross-chain swap to guarantee the transaction occurs on the two networks.
+
+1 – Alice and Bob issue a transaction on the Bitcoin and Litecoin blockchain.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: Prefer 1., 2., 3. notation for lists
See [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)

File: site/content/news/atomic-swaps.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+2 – Alice is able to claim the 20 LTC if she reveals a number that she only has access to.
+3 – Similarly, Bob is able to claim his BTC if he reveals the same number.
+4 – When one of them claim their tokens, the other one can do the same.
+
+(note : This is a simplified explanation of atomic swaps: the technology uses hash time-locked contracts, which are outside the scope of this article)  
+
+Previously, atomic swaps were considered to work on blockchains that have implement the Lightning Network.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: implemented

File: site/content/news/atomic-swaps.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+4 – When one of them claim their tokens, the other one can do the same.
+
+(note : This is a simplified explanation of atomic swaps: the technology uses hash time-locked contracts, which are outside the scope of this article)  
+
+Previously, atomic swaps were considered to work on blockchains that have implement the Lightning Network.  
+
+However, Charlie Lee, inventor of Litecoin and crypto guru, recently performed a swap between Litecoin / Decred and between Litecoin / Bitcoin without the need of the Lightning Network.  
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Yesterday I did an on-chain atomic swap of 1.337 LTC for 2.4066 DCR w/ <a href="https://twitter.com/_alyp_?ref_src=twsrc%5Etfw">@_alyp_</a> of <a href="https://twitter.com/decredproject?ref_src=twsrc%5Etfw">@decredproject</a>. (See txns: <a href="https://t.co/BlxU1QBK2U">https://t.co/BlxU1QBK2U</a>) ⛓️⚛️💱🚀 <a href="https://t.co/wPqzdw40Gp">https://t.co/wPqzdw40Gp</a></p>&mdash; Charlie Lee [NO2X] (@SatoshiLite) <a href="https://twitter.com/SatoshiLite/status/910534107058233344?ref_src=twsrc%5Etfw">September 20, 2017</a></blockquote>
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: Consider using Hugo's built-in shortcode for tweets:
{{< tweet 910534107058233344 >}} with the ID for the corresponding tweet
Simpler and looks better than pasting a bunch of code in the article

File: site/content/news/atomic-swaps.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Did a cross-chain atomic swap with LTC/BTC! 😁<br>10 LTC for 0.1137 BTC with <a href="https://twitter.com/JStefanop1?ref_src=twsrc%5Etfw">@JStefanop1</a>. ⛓️⚛️💱<a href="https://t.co/vXwTNirk0J">https://t.co/vXwTNirk0J</a><a href="https://t.co/3NTplBOoW9">https://t.co/3NTplBOoW9</a> <a href="https://t.co/DRKaHg4Wc7">pic.twitter.com/DRKaHg4Wc7</a></p>&mdash; Charlie Lee [NO2X] (@SatoshiLite) <a href="https://twitter.com/SatoshiLite/status/911328252928643072?ref_src=twsrc%5Etfw">September 22, 2017</a></blockquote>
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: Consider using the Hugo's built-in shortcode for tweets

File: site/content/news/atomic-swaps.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+It is worth noting that atomic swaps are currently at a development stage without any integration on a leading exchange, therefore it cannot cater mainstream cryptocurrency enthusiasts yet.  
+
+However, there are repositories available [here] (https://medium.com/decred/on-chain-atomic-swaps-bdb29fa2c92c) to conduct atomic swaps between Bitcoin, Litecoin and Decred.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: Incorrect link syntax

File: site/content/news/atomic-swaps.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+It is worth noting that atomic swaps are currently at a development stage without any integration on a leading exchange, therefore it cannot cater mainstream cryptocurrency enthusiasts yet.  
+
+However, there are repositories available [here] (https://medium.com/decred/on-chain-atomic-swaps-bdb29fa2c92c) to conduct atomic swaps between Bitcoin, Litecoin and Decred.  
+
+Atomic swaps only execute peer-to-peer trades and do not facilitate order book management and pricing. You will have to find a peer and negotiate with him before starting an atomic swap.  
+
+In fact, a fully-fledged decentralised trading solution would have to incorporate atomic swaps with an order book management and pleasant user experience before we can expect large scale trustless and peer-to-peer exchanges.  
+
+This is major breakthrough for the cryptocurrency world, as the publishing of open-source repositories enables users to issue transparent and private transactions and disintermediate the entire process.
```
Comment (alexisagos) [inline site/content/news/atomic-swaps.md]: a major breakthrough ?

File: site/content/news/snowden-zcash.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+weight = 5
+image = "snowden"
++++
+
+
+The famous NSA whistle-blower, Edward Snowden, started a massive debate amongst the cryptocurrency community when assessing two of the most popular privacy-focused tokens: ZCash and Monero.
+
+It all started when he tweet this bomb:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Agree. Zcash&#39;s privacy tech makes it the most interesting Bitcoin alternative. Bitcoin is great, but &quot;if it&#39;s not private, it&#39;s not safe.&quot; <a href="https://t.co/HqwQOvSCiz">https://t.co/HqwQOvSCiz</a></p>&mdash; Edward Snowden (@Snowden) <a href="https://twitter.com/Snowden/status/913544739542241282?ref_src=twsrc%5Etfw">September 28, 2017</a></blockquote>
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: prefer {{< tweet ID >}} syntax

File: site/content/news/snowden-zcash.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+draft = false
+description = "Famous NSA whistleblower took a endorsed Digital Currency Group's machinery and bashed Monero"
+weight = 5
+image = "snowden"
++++
+
+
+The famous NSA whistle-blower, Edward Snowden, started a massive debate amongst the cryptocurrency community when assessing two of the most popular privacy-focused tokens: ZCash and Monero.
+
+It all started when he tweet this bomb:
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: tweeted

File: site/content/news/snowden-zcash.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+In response to a tweet from Mason Borda which read: "Zcash is the only altcoin (that i know of) designed and built by professional and academic cryptographers. Hard to ignore".  
+Snowden suggested ZCash is the “most interesting Bitcoin alternative.”
+
+<blockquote class="twitter-tweet" data-conversation="none" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Great project, but the problem with amateur crypto is mistakes happen and have huge consequences for people like me: <a href="https://t.co/upyXId1Ubh">https://t.co/upyXId1Ubh</a></p>&mdash; Edward Snowden (@Snowden) <a href="https://twitter.com/Snowden/status/913557610858778625?ref_src=twsrc%5Etfw">September 29, 2017</a></blockquote>
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: prefer shortcode

File: site/content/news/snowden-zcash.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+
+Monero has been constantly improving their software to improve their privacy features. It has been disclosed in 2015 by the [Monero Research Lab](https://lab.getmonero.org/pubs/MRL-0004.pdf) that limitations existed concerning anonymity on their blockchain.  
+
+Interestingly, members of the board of the ZCash Foundation published an academic paper in 2017 reminding the vulnerabilities of Monero’s previous 2015 features on (MoneroLink)[http://monerolink.com/monerolink.pdf].  
+
+Mind that this website is run by ZCash stakeholders and has been promoted by CoinDesk and the Digital Currency Group.  
+
+Monero’s vulnerabilities have been addressed by the team till date. Back in 2015, Monero had only a few users and openly communicated about transaction linking discrepancies. 
+
+## Monero is nothing but amateur.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: I think you meant the opposite. This implies that monero is only amateur !
[See here](https://www.collinsdictionary.com/dictionary/english/nothing-but)

File: site/content/news/snowden-zcash.md [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+
+Achieving anonymity in cryptocurrency is not static and requires constant updates and improvement.  
+Stating that a cryptocurrency has the best privacy features should be relative to the other existing solutions.  
+
+While Monero should always keep on improving its solution, it is one of the best performing tokens in the market and therefore is nothing but amateur but rather paving the way for further improvement in this exciting space.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: Same error "nothing but"

File: site/content/news/snowden-zcash.md [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+## Monero is nothing but amateur.
+
+Achieving anonymity in cryptocurrency is not static and requires constant updates and improvement.  
+Stating that a cryptocurrency has the best privacy features should be relative to the other existing solutions.  
+
+While Monero should always keep on improving its solution, it is one of the best performing tokens in the market and therefore is nothing but amateur but rather paving the way for further improvement in this exciting space. 
+
+## A private company runs ZCash.
+
+ZCash has been introduced by a private firm ZCash Company, on the other hand, Monero is launched by a community of developers.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: was launched / is maintained. "is launched" implies it is being launched at present

File: site/content/news/snowden-zcash.md [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+While Monero should always keep on improving its solution, it is one of the best performing tokens in the market and therefore is nothing but amateur but rather paving the way for further improvement in this exciting space. 
+
+## A private company runs ZCash.
+
+ZCash has been introduced by a private firm ZCash Company, on the other hand, Monero is launched by a community of developers. 
+
+ZCash can be subpoenaed by the US government while Monero cannot.  
+A for-profit corporation handling a token does not seem fit to the decentralisation drive that blockchain technology is introducing.
+
+## Snowden did not address any of ZCash core issues.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: Zcash's core issues

File: site/content/news/snowden-zcash.md [hunk 18]
```diff
@@ -0,0 +1,89 @@
...
+
+Compared to Monero, ZCash token fundamentals are less attractive for users. As such, miners are taxed with a 20% fee from mining rewards disbursed to the founders for the next 4 years.  
+
+Furthermore, ZCash was set-up in closed doors in a trusted setup.  
+This means ZCash blockchain is vulnerable to a back door if the genesis block private keys were not destroyed as advertised.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: Zcash's blockchain

File: site/content/news/snowden-zcash.md [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+A for-profit corporation handling a token does not seem fit to the decentralisation drive that blockchain technology is introducing.
+
+## Snowden did not address any of ZCash core issues.
+
+Compared to Monero, ZCash token fundamentals are less attractive for users. As such, miners are taxed with a 20% fee from mining rewards disbursed to the founders for the next 4 years.  
+
+Furthermore, ZCash was set-up in closed doors in a trusted setup.  
+This means ZCash blockchain is vulnerable to a back door if the genesis block private keys were not destroyed as advertised.  
+ZCash token holders will always have to trust the founding team that nothing adverse happened during the token introduction, without any hard evidence to prove it.  
+In other words, there is no guarantee the US government or any other third party has a backdoor.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: Opposite meaning ! "that the US government or any other third party doesn't have a backdoor."

File: site/content/news/snowden-zcash.md [hunk 20]
```diff
@@ -0,0 +1,89 @@
...
+Compared to Monero, ZCash token fundamentals are less attractive for users. As such, miners are taxed with a 20% fee from mining rewards disbursed to the founders for the next 4 years.  
+
+Furthermore, ZCash was set-up in closed doors in a trusted setup.  
+This means ZCash blockchain is vulnerable to a back door if the genesis block private keys were not destroyed as advertised.  
+ZCash token holders will always have to trust the founding team that nothing adverse happened during the token introduction, without any hard evidence to prove it.  
+In other words, there is no guarantee the US government or any other third party has a backdoor. 
+
+## CoinDesk and ZCash are owned by the same fund.
+
+The Digital Currency Group has a stake in both the cryptocurrency ZCash and the media outlet CoinDesk. CoinDesk has therefore an agenda to promote ZCash against its main competition Monero.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: competitor ?

File: site/content/news/snowden-zcash.md [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+Furthermore, ZCash was set-up in closed doors in a trusted setup.  
+This means ZCash blockchain is vulnerable to a back door if the genesis block private keys were not destroyed as advertised.  
+ZCash token holders will always have to trust the founding team that nothing adverse happened during the token introduction, without any hard evidence to prove it.  
+In other words, there is no guarantee the US government or any other third party has a backdoor. 
+
+## CoinDesk and ZCash are owned by the same fund.
+
+The Digital Currency Group has a stake in both the cryptocurrency ZCash and the media outlet CoinDesk. CoinDesk has therefore an agenda to promote ZCash against its main competition Monero.  
+
+Here, Snowden linked a CoinDesk article to both promote ZCash’s tech features and qualify Monero as “amateur” crypto. Further to that the Monero vulnerabilities stated in the article are outdated.
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: "Further to that the Monero vulnerabilities" missing comma or word

File: site/content/news/snowden-zcash.md [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+
+However, his open endorsement to ZCash while bashing Monero by sharing a link to a biased media house can be quite controversial and might suggest Snowden is not up to date with cryptocurrency technology or has an agenda promote Digital Currency Group’s products and services.  
+
+This might just jeopardize Snowden’s credibility as a security expert… at least in the cryptocurrency space.
+
+Read our privacy focused-token review [here](/news/privacy-focused-tokens)
+
+FYI: Litecoin’s founder Charlie Lee owns Monero and not ZCash.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I own Monero but not Zcash</p>&mdash; Charlie Lee [NO2X] (@SatoshiLite) <a href="https://twitter.com/SatoshiLite/status/913605887247257602?ref_src=twsrc%5Etfw">September 29, 2017</a></blockquote>
```
Comment (alexisagos) [inline site/content/news/snowden-zcash.md]: prefer shortcode

File: site/content/news/what-the-fork.md [hunk 23]
```diff
@@ -107,7 +115,10 @@ Because of this uncertainty, miners designed a contingency plan to mitigate the
 ### User Activated Hard Fork (UAHF)
 
 Introduced by Bitmain, a Bitcoin mining pool, UAHF is the adoption of a new Bitcoin chain with an increased block size and without the implementation of SegWit.  
-This proposal does not require any minimum hashing power from the network and any nodes that accept the underlying condition of UAHF is able to interact with the network. UAHF is an alternative to Segwit as some stakeholders were not satisfied with the reduction of mining incentivisation and the separation of signatures from transaction data.  
+
+This proposal does not require any minimum hashing power from the network and any nodes that accept the underlying condition of UAHF is able to interact with the network.
```
Comment (alexisagos) [inline site/content/news/what-the-fork.md]: any **node** that **accepts** the underlying condition of UAHF is able to interact with the network.

**Comment by cthepot** [comment]
new infographics have slight changes. Bob has a shorter one now

**Pull request merged**